<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="tabsWidget">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="view_settings">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox__edges">
          <property name="text">
           <string>Edges</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_vertices">
          <property name="text">
           <string>Vertices</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="info">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="file_info">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>250</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="EOpenGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenModel"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenModel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tabs">
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMove_Rotate"/>
   <addaction name="actionTabSettings"/>
   <addaction name="actionTabInfo"/>
  </widget>
  <action name="actionOpenModel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/Folder_open_alt_font_awesome.svg.png</normaloff>:/icons/icons/Folder_open_alt_font_awesome.svg.png</iconset>
   </property>
   <property name="text">
    <string>Open Model</string>
   </property>
  </action>
  <action name="actionTabSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/2040504.png</normaloff>:/icons/icons/2040504.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTabInfo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/info-icon-png-transparent-20.jpeg</normaloff>:/icons/icons/info-icon-png-transparent-20.jpeg</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Info</string>
   </property>
  </action>
  <action name="actionMove_Rotate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/3d-modeling-icon.webp</normaloff>:/icons/icons/3d-modeling-icon.webp</iconset>
   </property>
   <property name="text">
    <string>Move/Rotate</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>eopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
